lib/fog/core/associations/many_identities.rb

```diff
@@ -19,7 +19,7 @@ def #{name}=(new_#{name})
       def create_getter
         model.class_eval <<-EOS, __FILE__, __LINE__
           def #{name}
-            return [] if associations[:#{name}].nil?
+            return [] if associations.empty? || associations[:#{name}].nil?
```

Comment:
I think I understand the other changes, but I'm less sure about this one. Could you elaborate?

Reply:
Hmmmm. I guess you are right. I was trying to solve another issue, but this code changes nothing.

For some reason it was returning nil when the association did not exist when it should return an empty array because it is a collection. Will remove this change.